web-registry/src/lib/env.ts

```diff
@@ -1 +1,2 @@
 export const IS_DEV = process.env.NODE_ENV === 'development';
+export const apiServerUrl = process.env.REACT_APP_API_URI;
```

Comment:
i think i remember Marie was referencing this as "apiUri" or something like that, there might be a small conflict there..